server/py/services/api/api/endpoints/model_endpoints.py

```diff
@@ -546,6 +590,29 @@ class _MetricsValuesParams:
     end: datetime
 
 
+def _validate_time_range(
```

Comment:
please have a look at the helper func `get_start_end`. I think you can use it instead of this new `_validate_time_range` and to avoid code duplication. 
I saw that you are calling it from two places, in both of them I think you can replace 
`start, end = _validate_time_range(start, end)`
with 
`start, end = mlrun.model_monitoring.helpers.get_start_end(start, end, delta=timedelta(days=1))`